pos-print/src/main/java/com/clouway/pos/print/adapter/db/PersistentCashRegisterRepository.kt

```diff
@@ -56,19 +56,31 @@ class PersistentCashRegisterRepository @Inject constructor(private val database:
   }
 
   private fun adapt(record: Document): CashRegister {
+    val policy = record["fiscalPolicy"] as Document
+    val policies = when {
+      policy.isEmpty() -> mutableListOf(FiscalPolicy(0, 20.0))
```

Comment:
isNullOrEmpty as the document may be null.

Reply:
fixed